api/src/main/java/org/apache/iceberg/actions/ActionsProvider.java

```diff
@@ -67,4 +67,12 @@ default RewriteDataFiles rewriteDataFiles(Table table) {
   default ExpireSnapshots expireSnapshots(Table table) {
     throw new UnsupportedOperationException(this.getClass().getName() + " does not implement expireSnapshots");
   }
+
+  /**
+   * Instantiates an action to remove all the files reachable from given metadata location.
+   */
+  default RemoveReachableFiles removeReachableFiles(String metadataLocation) {
+    throw new UnsupportedOperationException(this.getClass().getName() + " does not implement " +
+        RemoveReachableFiles.class.toString());
```

Comment:
nit: the others use the method name in the api and not the class name of the api

Reply:
+1